users.md

```diff
@@ -58,14 +58,12 @@ title: Users
 </div>
 {% endfor %}
 
-<div class="row spacer-30">
-  <div class="row">
-    <div class="col-md-12 col-s-12">
-    <h1>Are you a user too?</h1>
-    <p>We are happy to include your testimonial! Contact us on the
-      <a href="./slack.html">community chat</a>, and we will get you added.</p>
-    </div>
-  </div>
-</div>
+<div markdown="1" class="row spacer-60 col-md-12 leftcol widecol">
 
+# Are you a user too?
+
+We are happy to include your testimonial! Contact us on the
+[community chat](slack.html), and we will get you added.
```

Comment:
i believe the link should be `/slack.html`, but that's pre-existing anyway. No change requested.

Reply:
Both should work. ./slack.html would work as well.

Reply:
It works when the page is loaded as `/users.html`, but won't work if you do `/users` since Netlify will redirect to `/users/` and then it's in the wrong "directory". But this is pre-existing and we need to fix this everywhere. We should see if Jekyll has a better way to link to things.

@mosabua what is the benefit of `./foo` vs `foo`? That only seems relevant in a shell where you have a search path.

Reply:
I wrong wrong in two ways:

1. It is rendered as `/slack.html` in the HTML, so it seems Jekyll is being smart here.
2. Netlify does not redirect [trailing slash](https://docs.netlify.com/routing/redirects/redirect-options/#trailing-slash) since we don't have "Pretty URLs" enabled. I'd like to do that, but it will break the documentation, since that is proxied from GitHub Pages and thus won't be rewritten to work properly. We don't host it on Netlify since the sheer number of files due to all the historical documentation was breaking Netlify deploys.